packages/lodestar-beacon-state-transition/src/block/operations/deposit.ts

```diff
@@ -2,7 +2,7 @@
  * @module chain/stateTransition/block
  */
 
-import {verify} from "@chainsafe/bls";
+import bls from "@chainsafe/bls";
```

Comment:
To have a globally available bls object that gets set conditionally named exports didn't work. Instead it exports a bls variable that gets mutated on init()
https://github.com/ChainSafe/bls/blob/c8ddfcaa3b12bdc81557686958e36b07535f42c3/src/index.ts#L9

Reply:
Using a specific implementation directly allows the use a named exports
```ts
import {verify} from "@chainsafe/bls/browser"
```

Reply:
oh man, than we made bls worse. And it sucks in  web usage unless you import from /browser.
I guess that was big arg for my way of abstracting it. :sob:

Reply:
In your abstraction you never got to add blst and herumi in the same PR. Only abstracted herumi without BLST. I'm down to change the abstraction if it works well on all settings!

Reply:
we just needed to implement given interface with blst and it would work...

Reply:
> we just needed to implement given interface with blst and it would work 
I'm not so sure about that. Again, if you know a way to make it work I would love to merge it

> Why haven't you raised concern that you failed to implement named export?
The current export strategy was there from the first iteration of the PR

Reply:
Also you don't have to use default exports, you can do
```ts
import {bls} from "@chainsafe/bls";
```
And it would work as well

Reply:
resolved w chainsafe/bls#44, will update here